### DISCUSSION TITLE
What does the Ctrl + L shortcut do?

### DISCUSSION THREAD

**Discussion by jamesshenry** [VSCodeVim/Vim#7390] [Q&A]
I have [dendron](https://www.dendron.so/) installed, but its default Ctrl+L shortcut is being overridden by VsCodeVim's. 
The vim command is extension.vim_navigateCtrlL but I can't figure out what it actually does...
```json
{
  "key": "ctrl+l",
  "command": "extension.vim_navigateCtrlL",
  "when": "editorTextFocus && vim.active && vim.use<C-l> && !inDebugRepl"
}
```

**Comment by kola-web** [comment]
see https://github.com/VSCodeVim/Vim/blob/master/ROADMAP.md

**Reply by jamesshenry** [reply]
Perhaps I am missing something but there is no CTRL-L command in this list

**Reply by ScriptAutomate** [reply]
I'm also confused here, as `CTRL-L` is not listed on the roadmap page

**Comment by damccull** [comment]
Nobody knows after 2 years? I just ran into this.

**Comment by Edgar-P-yan** [comment]
Looks like `ctrl-l` only works in `SearchInProgress` mode: https://github.com/VSCodeVim/Vim/blob/cd8afe4d625c3013ed56c54b93be4fb6e4f2de80/src/actions/commands/commandLine.ts#L384

While typing the search term the `ctrl-l` will autocomplete the query by one letter from the first match.
Try it out, type `/str` in normal mode. Then press `ctrl-l`, it'll add one more letter to the search from the first match, probably the letter `i`, press one more time and it'll add `n` and so on, untill it has no more letters for that autocompletion.
<img width="507" alt="image" src="https://github.com/user-attachments/assets/fd0c2448-8764-4f1d-80df-7fce51ce9b82">

**Comment by janreggie** [comment]
If anyone encountered this thread and wants to set Ctrl+L to lookup notes even when Vim is active, please add the following to your user keyboard shortcuts:

```json
{
  "key": "ctrl+l",
  "command": "dendron.lookupNote",
  "when": "dendron:pluginActive"
}
```
